fluentui_persona/src/test/java/tokenized/persona/InitialsTest.kt

```diff
@@ -0,0 +1,108 @@
+
+class InitialsTest {
+
+    companion object {
+        private val person = Person("First", "Last", "firstlast@example.com")
+        private val firstNamePerson = Person("First", email = "firstlast@example.com")
+        private val personWithDigit = Person("First", "2nd", "firstthe2nd@example.com")
+        private val personWithSpecialChar = Person("_First", "@Last", email = "firstthe2nd@example.com")
+        private val noNamePerson = Person("", email = "firstlast@example.com")
+        private val noNameNoEmailPerson = Person("", email = "")
+        private val group = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "Gang Of 4")
+        private val groupNoNameNoEmail = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "")
+        private val groupNoName = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "", email = "4@example.com")
+        private val groupDigit = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "4")
+        private val groupAlphaNumeric = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "4 People")
+        private val groupSingleName = Group(listOf(person, personWithDigit, personWithSpecialChar, noNamePerson), groupName = "Four")
+    }
+
+    @Test
+    fun testGetInitials(){
+        val expectedInitials = "FL"
+        val resultantInitials = person.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetInitialsFirstNameOnly(){
+        val expectedInitials = "F"
+        val resultantInitials = firstNamePerson.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetInitialsWithDigits(){
+        val expectedInitials = "F2"
+        val resultantInitials = personWithDigit.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetInitialsWithSpecialChars(){
+        val expectedInitials = "FL"
+        val resultantInitials = personWithSpecialChar.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetInitialsNoName(){
+        val expectedInitials = "F"
+        val resultantInitials = noNamePerson.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetInitialsNoNameNoEmail(){
+        val expectedInitials = "#"
+        val resultantInitials = noNameNoEmailPerson.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetGroupInitials() {
+        val expectedInitials = "GO"
+        val resultantInitials = group.getInitials()
+        Assert.assertEquals(expectedInitials, resultantInitials)
+    }
+
+    @Test
+    fun testGetGroupInitialsNoNameNoEmail() {
```

Comment:
In this case, getInitials() expected to return empty string ?

Reply:
similar as above